lib/modules/@apostrophecms/piece-type/src/apos/components/AposPiecesManager.vue

```diff
@@ -0,0 +1,254 @@
+            <tbody>
+              <tr>
+                <th class="apos-table__header" />
+                <th
+                  v-for="header in headers" scope="col"
+                  class="apos-table__header" :key="header.label"
+                >
+                  <component
+                    :is="getEl(header)" @click="sort(header.action)"
+                    class="apos-table__header-label"
+                  >
+                    <component
+                      v-if="header.icon"
+                      :size="iconSize(header)"
+                      class="apos-table__header-icon"
+                      :is="icons[header.icon]"
+                    />
+                    {{ header.label }}
+                  </component>
+                </th>
+              </tr>
+              <tr
+                class="apos-table__row"
+                v-for="row in rows"
+                :key="row._id"
+                :class="{'is-selected': false }"
+              >
+                <td class="apos-table__cell">
+                  <AposCheckbox
+                    v-if="checkboxes[row.id]"
+                    :field="checkboxes[row.id].field"
+                    :value="checkboxes[row.id].value.data"
+                    :status="checkboxes[row.id].status"
+                    :choice="checkboxes[row.id].choice"
+                    :id="row.id"
+                    v-model="checked"
+                  />
+                </td>
+                <td
+                  class="apos-table__cell" v-for="header in headers"
+                  :key="row[header.name]"
+                >
+                  <a
+                    v-if="header.name === 'url'" class="apos-table__link"
+                    :href="row[header.name]"
+                  >
+                    <LinkIcon :size="12" />
+                  </a>
+                  <p
+                    v-else class="apos-table__cell-field"
+                    :class="`apos-table__cell-field--${header.name}`"
+                  >
+                    {{ row[header.name] }}
+                  </p>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <div v-else class="apos-pieces-manager__empty">
+            <AposEmptyState :empty-state="emptyDisplay" />
+          </div>
+        </template>
+      </AposModalBody>
+      <!-- The pieces editor modal. -->
+      <component
+        v-if="inserting" :module-name="moduleName"
+        :is="options.components.insertModal" @close="inserting = false"
+        @saved="finishSaved"
+      />
+    </template>
+  </AposModal>
+</template>
+
+<script>
+import AposTableMixin from '../../../../modals/mixins/AposTableMixin';
+import AposModalParentMixin from '../../../../modals/mixins/AposModalParentMixin';
+
+export default {
+  name: 'AposPiecesManager',
+  mixins: [ AposTableMixin, AposModalParentMixin ],
+  props: {
+    moduleName: {
+      type: String,
+      required: true
+    }
```

Comment:
We're no longer getting passed the `tagList` or `applyTags`. Those will need to be done in here somewhere... Are we still going to have tags in the pieces manager?